src/common/ceph_context.h

```diff
@@ -42,12 +43,15 @@
 #include "crush/CrushLocation.h"
 
 class AdminSocket;
-class CephContextServiceThread;
-class CephContextHook;
-class CephContextObs;
 class CryptoHandler;
 class CryptoRandom;
 
+namespace TOPNSPC::common {
+  class CephContextServiceThread;
```

Comment:
this class is defined in `ceph::common`, see `ceph_context.cc`. or am i missing something?